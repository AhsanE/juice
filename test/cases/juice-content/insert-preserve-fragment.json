{
    "preserveFontFaces": true,
    "preserveMediaQueries": true,
    "insertPreservedExtraCss": true,
    "extraCss": "@font-face { font-family: 'Extra'; }"
}
