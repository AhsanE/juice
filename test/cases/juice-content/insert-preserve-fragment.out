<div></div>
<style>
@font-face {
    font-family: 'Extra';
}
</style>
